### ISSUE TITLE
sequence_rma_item error when migrating ce-to-ce 2.0.5+

### ISSUE THREAD

**Issue by flecxie** [magento/data-migration-tool#107]
Started getting this error after upgrading from magento 2.0.4 to 2.0.7 (CE) and running data-migration from a magento 1.7.0.2 server:

`[2016-07-05 05:06:17][ERROR]: Destination documents are not mapped: sequence_rma_item_0,sequence_rma_item_1,sequence_rma_item_2,sequence_rma_item_3,sequence_rma_item_4,sequence_rma_item_5,sequence_rma_item_6,sequence_rma_item_7,sequence_rma_item_8`

Didn't get this error before so seems to be some new field in CE 2.0.5 (see also report from a user on the Magento community forum: https://community.magento.com/t5/Technical-Issues/Issues-after-Migrating-from-1-9-2-1-to-Magento-2-0-5/td-p/37356). 

Error went away after applying the following patch to my map.xml:
```
                 <document>sequence_shipment*</document>
             </ignore>
             <ignore>
+                <document>sequence_rma_item_*</document>
+            </ignore>
+            <ignore>
                 <document>eav_attribute_option_swatch</document>
             </ignore>
             <ignore>
```

**Comment by victor-v-rad** [comment]
Hi @flecxie 
Thank you for posting this issue. It was fixed in the new Data Migration Tool 2.1

**Comment by flecxie** [comment]
Thanks @victor-v-rad!
